app/controllers/simplified-account/settings/email-notifications/email-notifications.controller.js

```diff
@@ -1,19 +1,51 @@
 const { response } = require('../../../../utils/response')
 const humaniseEmailMode = require('../../../../utils/humanise-email-mode')
+const formatSimplifiedAccountPathsFor = require('../../../../utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('../../../../paths')
+const { setEmailCollectionModeByServiceIdAndAccountType } = require('../../../../services/email.service')
+const logger = require('../../../../utils/logger')(__filename)
 
-function get (req, res) {
+function getEmailNotificationsSettingsPage (req, res) {
```

Comment:
the controller is already named `email-notifications`. `emailNotifications.getEmailNotificationsSettingsPage` is not any clearer (and is significantly more verbose) than `emailNotifications.get` imo

Reply:
I mentioned on [slack](https://gds.slack.com/archives/C03B49R80NA/p1730197373310689?thread_ts=1730194963.922309&cid=C03B49R80NA) I don't really like just the "get" as it's non-descriptive.